api/apps/users/model.coffee

```diff
@@ -78,19 +75,27 @@ save = (user, accessToken, callback) ->
     }, callback
 
 #
+# Utility
+#
+@hasChannelAccess = (user, channel_id) ->
```

Comment:
👍  nice extraction of logic here— a little much on the newline whitespace for my taste, but I don't claim to have much of a refined palette anyway.